### PR TITLE
Updated setup and Introduction to refer to Apptainer, to expand instr…

### LINKED ISSUE #6: Add: Pros/cons/Differences/Similarities between docker/singularity

### LINKED ISSUE #27: Discussion: How to deal with Singularity vs Apptainer
While the training module is still called Singularity, it should be mentioned in the introduction that **Apptainer** is now the name of the project included in the Linux Foundation. 

It is relevant to mention Compatibility between Apptainer and Singularity:
https://apptainer.org/docs/user/main/singularity_compatibility.html

### PR DESCRIPTION
**Pull request by mambelli**
Updated setup and Introduction to refer to Apptainer, expand instructions, and clarify differences between Apptainer, Docker, and other Singularity versions.

This work was part of the 2022-01-06 hackathon:
* Expand the setup, issue #2 (I referred to existing instructions)
* Add: Pros/cons/Differences/Similarities between docker/singularity, Fixes #6 
* Singularity or Apptainer?, Fixes #27 - before the end of September Apptainer is expected to replace Singularity in EPEL. At a later time, we'll have to think about switching the command name and environment variables.

### CODE REVIEW

**Comment by amanmdesai** [comment]
pre-commit failure is due to this

_episodes/01-introduction.md:47: withot ==> without
setup.md:6: deatils ==> details

**Comment by mambelli** [comment]
> pre-commit failure is due to this
> 
> _episodes/01-introduction.md:47: withot ==> without setup.md:6: deatils ==> details

Fixed, thanks

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -27,32 +28,55 @@ environment used at that time.
 * All the necessary packages to process data can be easily distributed in containers, independently of the operating
 system available on the sites.
 
-# Why Singularity?
+Virtual Machines (VMs) provide the same isolation and reproducibility but emulate the hardware, everything in the computer,
+so they are computationally heavier to run, require bigger files when distributed and are less flexible then containers,
```
Comment (michmx) [inline _episodes/01-introduction.md]: typo: then --> than

**Comment by michmx** [comment]
Thanks a lot, @mambelli ! Very nice explanation of Singularity --> Apptainer. 

I just found a typo. Besides that, this PR is ready to go.

**Review by michmx — CHANGES_REQUESTED** [review]

File: _episodes/01-introduction.md (hunk 1 above)
Comment (mambelli) [reply _episodes/01-introduction.md]: updated, thanks

**Review by michmx — APPROVED** [review]

**Pull request merged**